github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.37.yaml

```diff
@@ -0,0 +1,612 @@
+periodics: null
+postsubmits: {}
+presubmits:
+  kubevirt/kubevirt:
+  - always_run: false
+    branches:
+    - release-0.37
+    context: pull-kubevirt-e2e-k8s-1.20
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 7h0m0s
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-shared-images: "true"
+    max_concurrency: 11
+    name: pull-kubevirt-e2e-k8s-1.20-0.37
```

Comment:
not that it matters much, but shouldn't this be `pull-kubevirt-e2e-k8s-1.20-release-0.37`? Or is the length a problem?

Reply:
Hm, it follows the usual `config-forker` convention of replacing `-master` with `-version`. Only the branches have the `release-` as prefix.